### PR TITLE
Fix constant mapping in J9ConstantPoolCommand

### PR DESCRIPTION
**Pull request by fengxue-IS**
Signed-off-by: Jack Lu <Jack.S.Lu@ibm.com>

### CODE REVIEW

**Comment by fengxue-IS** [comment]
@tajila can you take a look please

**Comment by tajila** [comment]
jenkins test sanity alinux64 jdk17

**Review by tajila — APPROVED** [review]

File: debugtools/DDR_VM/src/com/ibm/j9ddr/CompatibilityConstants29.dat [hunk 1]
```diff
@@ -69,8 +69,9 @@ J9JavaClassFlags.J9ClassRequiresPrePadding = 0
 J9NonbuilderConstants.J9_ROMCLASS_OPTINFO_INJECTED_INTERFACE_INFO = 0
 J9NonbuilderConstants.J9_ROMCLASS_OPTINFO_PERMITTEDSUBCLASSES_ATTRIBUTE = 0
 J9NonbuilderConstants.J9_ROMCLASS_OPTINFO_RECORD_ATTRIBUTE = 0
-J9NonbuilderConstants.J9DescriptionCpBsmIndexMask = 0
-J9NonbuilderConstants.J9DescriptionCpTypeShift = 0
+
+J9DescriptionBits.J9DescriptionCpBsmIndexMask = 0
+J9DescriptionBits.J9DescriptionCpTypeShift = 0
```
[the remark this answers is not in the document]
Comment (keithc-ca) [reply debugtools/DDR_VM/src/com/ibm/j9ddr/CompatibilityConstants29.dat:74]: Correction: only `J9DescriptionCpBsmIndexMask` is needed here.

[the remark this answers is not in the document]
Comment (keithc-ca) [reply debugtools/DDR_VM/src/com/ibm/j9ddr/CompatibilityConstants29.dat:74]: I created https://github.com/eclipse-openj9/openj9/pull/17376 to remove address this.

[the remark this answers is not in the document]
Comment (fengxue-IS) [reply debugtools/DDR_VM/src/com/ibm/j9ddr/CompatibilityConstants29.dat:74]: Didnt realize CPTypeShift is already available, thanks for fixing Keith.

**Pull request merged**